binder/summary-statistics.ipynb

```diff
@@ -0,0 +1,110 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from keyname import keyname as kn\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "import seaborn as sns\n",
+    "from teeplot import teeplot\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv(\"https://osf.io/ck47r/download\")\n"
```

Comment:
add logging details like

```


# prints metadata about notebook runtime
nbm.print_metadata()

df = pd.read_csv("https://osf.io/5trmg/download")
dfdigest = np.bitwise_xor.reduce(
    pd.util.hash_pandas_object(df),
)
print("{:x}".format(dfdigest))
df
```

from https://github.com/mmore500/hstrat-evolutionary-inference/blob/cc6472840ef97e0edacc83fb4d5cf139b22a6c84/binder/perfect-tree-phylometrics.ipynb

Reply:
Done in 589fbbc